pkgs/tools/networking/pixiewps/default.nix

```diff
@@ -2,18 +2,16 @@
 
 stdenv.mkDerivation rec {
   pname = "pixiewps";
-  version = "1.2.2";
+  version = "1.4.2";
   src = fetchFromGitHub {
-    owner = "wiire";
+    owner = "wiire-a";
```

Comment:
The homepage should also be updated with the new username (with a small comment in the commit that the owner changed its username)

Reply:
Hello,

sorry I messed up trying to amend a commit and accidentally had this PR closed. Could you please take a look to the [new PR](https://github.com/NixOS/nixpkgs/pull/184201)?